src/screens/EnterPrivateKey/EnterPrivateKeyStyles.js

```diff
@@ -0,0 +1,11 @@
+import { StyleSheet } from 'react-native'
+import {Header} from 'react-navigation'
```

Comment:
According to the information from official react-navigation web site, this is temporary "solution". So, we should not use Header.HEIGHT.